### PR TITLE
Optimize CI a bit

### PR DESCRIPTION
**Pull request by Goooler**

### CODE REVIEW

File: .github/workflows/publish_release.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: github.repository == 'usefulness/easylauncher-gradle-plugin'
```
Comment (Goooler) [inline .github/workflows/publish_release.yml:11]: See https://github.com/square/kotlinpoet/pull/1119.

**Comment by Goooler** [comment]
```
Run actions/setup-java@v3
Trying to resolve the latest version from remote
Error: Could not find satisfied version for SemVer '15'. 
Available versions: 18.0.0+36, 17.0.2+8, 17.0.1+12, 17.0.0+35, 16.0.2+7, 11.0.14+101, 11.0.14+9, 11.0.[13](https://github.com/usefulness/easylauncher-gradle-plugin/runs/5954848046?check_suite_focus=true#step:3:13)+8, 11.0.12+7, 8.0.322+6, 8.0.312+7, 8.0.302+8
```

**Comment by Goooler** [comment]
May we need to use `java 17` or `zulu`?

**Review by mateuszkwiecinski — APPROVED** [review]
Thanks! And yes, Java 17 is fine 👌

**Pull request merged**